### ISSUE TITLE
Doesn't generate some images

### ISSUE THREAD

**Issue by Duskfall** [sindresorhus/gulp-imagemin#201]
Hello, I have a problem with the output of imagemin.
I use latest version of gulp-imagemin

````
var images = [
        './Content/Images/**/*.*',
        './Content/Images/*.*',
        './PdfJs/web/images/*.*'
];
var  imagesOut = './dist/images';

gulp.task('images-min', ['clean', 'bower-restore'], function () {
    return gulp.src(config.images)
      .pipe(imagemin({ optimizationLevel: 3, progressive: true, interlaced: true }))
      .pipe(gulp.dest(config.imagesOut));
});

Only some images are copied and not all. Same happens with default configuration.

**Comment by sindresorhus** [comment]
Make sure you're on the latest version. There was a related fix in 3.0.2.

**Comment by MasterOfZozo** [comment]
Your configuration look like a you have version 2. Try update to 3.0.2. You will only need rewrite your config.

I recommend something like this:
```
let plugins = [
    imagemin.gifsicle({
      interlaced: false, //lossy
      optimizationLevel: 3
    }),
    imagemin.jpegtran({
      progressive: false, //lossy
      arithmetic: true //better compression
    }),
    imagemin.optipng({
      optimizationLevel: 3 //0-7, higher means better compression and lower performance
    }),
    imagemin.svgo({
      multipass: true, // best compression
      svgoPlugins: [
        {cleanupIDs: false}, //unused by svg file, but what about website?
        {removeViewBox: false}, // IE and Edge require view box
        {removeUselessStrokeAndFill: false}, // are strokes and fill really unuseful?
        {removeEmptyAttrs: false}, // allowed by HTML5
        {
          removeAttrs: {
            attrs: ["xmlns"]
          }
        }
      ]
    })];
  let options = {
    verbose: true
  };
  gulp.src(images.files.src)
          .pipe(plumber({errorHandler: error.console}))
          .pipe(changed(images.folder.dest))
          .pipe(imagemin(plugins, options))
          .pipe(gulp.dest(images.folder.dest));
```
